src/test/java/api/loans/scenarios/ChangeDueDateAPITests.java

```diff
@@ -377,7 +379,7 @@ void dueDateChangeShouldClearRenewalFlagWhenSetAndNoOpenRecallsInQueue() {
     ZonedDateTime recalledLoanDueDate = ZonedDateTime.parse(recalledLoan.getJson().getString("dueDate"));
 
     assertThat(recalledLoan.getJson().getBoolean("dueDateChangedByRecall"), equalTo(true));
-    assertThat(calculateDaysBetween(initialDueDate, recalledLoanDueDate), equalTo(16));
+    assertThat(calculateDaysBetween(initialDueDate, recalledLoanDueDate), equalTo(4));
```

Comment:
I'm afraid it might not work on a day near the end of the month again...

Reply:
i don't think so, since rolling is based on weeks and it is fixed to 7 days, not like month which has alternative one

Reply:
fixed